### PR TITLE
Niyologo

### PR DESCRIPTION
**Pull request by niyolynate**

### CODE REVIEW

**Comment by Ismael-KG** [comment]
Hi @niyolynate! There seems to be a technical problem with this PR: you need to close it and make another _without_ a submodule. If you're struggling with this, please ask in the branding channel on Slack 😄
image

**Comment by niyolynate** [comment]
@Ismael-KG thanks alot, i really want to closing it and failed. let me try again.

**Comment by Ismael-KG** [comment]
Let me know if you can't see the "Close with comment" at the bottom right like me

Screenshot 2022-10-26 at 15 24 43

**Comment by niyolynate** [comment]
i actually want to delete the whole PR. if possible. that branch would be named differently. Please @Ismael-KG  help me have failed to locate the delete button.. whole PR.

**Comment by Ismael-KG** [comment]
@niyolynate, you'll need to close it first, anyway. I think the option to delete the branch appears afterwards (see [this GitHub guide](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-branches-in-your-repository/deleting-and-restoring-branches-in-a-pull-request)).

**Comment by Bisola15** [comment]
Hi @niyolynate. You can delete the branch directly from your repo too. If you want I can take you through it over a slack huddle

**Comment by niyolynate** [comment]
ok @Bisola15   let m try that thanks
